cmd/snap-seccomp/main.go

```diff
@@ -19,6 +19,10 @@
 
 package main
 
+//#cgo pkg-config: --static --libs libseccomp
```

Comment:
Ironically this is incorrect as you'd actually link with libseccomp *dynamically*. This outputs `-lseccomp` which is added to the compiler. The line below hard-codes that same (effective) library between two linker options that switch link mode. I think we should drop the pkg-config line and just keep the LDFLAGS line below.

I'll test this locally and push if you agree.